File: internal/adapter/driving/hyperdeck/codes.go
// Package hyperdeck implements the HyperDeck Ethernet Protocol as a driving adapter.
package hyperdeck

// Response codes (HyperDeck Ethernet Protocol v1.11). Verify against the manual.
const (
	CodeOK             = 200
	CodeSlotInfo       = 202
	CodeDeviceInfo     = 204
	CodeClipsInfo      = 205
	CodeTransportInfo  = 208
	CodeConnectionInfo = 500
	CodeSyntaxError    = 100
	CodeInvalidState   = 150
)

File: crates/hyperdeck-core/src/protocol/codes.rs
//! HyperDeck Ethernet Protocol response codes (v1.11).
//! Port of `internal/adapter/driving/hyperdeck/codes.go`.

pub const CODE_OK: u16 = 200;
pub const CODE_SLOT_INFO: u16 = 202;
pub const CODE_DEVICE_INFO: u16 = 204;
pub const CODE_CLIPS_INFO: u16 = 205;
pub const CODE_TRANSPORT_INFO: u16 = 208;
pub const CODE_CONNECTION_INFO: u16 = 500;
pub const CODE_SYNTAX_ERROR: u16 = 100;
pub const CODE_INVALID_STATE: u16 = 150;
